desktop/dev-environments.md

```diff
@@ -43,6 +43,10 @@ If it doesn't detect Git as a valid command, you must reinstall Git and ensure y
 
 ![Windows add Git to path](images/dev-env-gitbash.png){:width="300px"}
 
+> **Note**
+>
+> You might need to quit and restart entirely Docker Desktop once git has been installed. From the Docker menu, select **Quit Docker Desktop**, then restart it.
```

Comment:
```suggestion
> You might need to quit and restart entirely Docker Desktop once git has been installed. From the Docker menu, select **Restart**, then restart it.
```

Not sure if it's "Restart" on windows too

Reply:
Just checked - it's `Restart Docker...` on Windows

Reply:
Actually no, "Restart Docker..." will continue execute in the same process and not catch new entries added in the PATH after Docker Desktop has been started. (Checked, the Dashboard window gets refreshed but stays alive when clicking "Restart Docker")

Reply:
How about "After you have installed Git, you must quit and then start Docker Desktop. From the Docker menu, select **Quit Docker Desktop**, and then start the application."?